includes/application/AskPostsOptions.php

```diff
@@ -55,7 +55,7 @@
 
 while($row = mysqli_fetch_row($res))
 {
-	echo "<option value=$row[0]>$row[0]</option>";
+	echo "<option value=\"$row[0]\">$row[0]</option>";
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed

<!-- Comment by [SideCI](https://sideci.com) -->